packages/nx/src/hasher/task-hasher.ts

```diff
@@ -437,6 +435,14 @@ class TaskHasherImpl {
     return partialHashes;
   }
 
+  private hashFiles(files: string[]): Map<string, string> {
```

Comment:
Let's rename this to `hashOutputFile` in a followup PR so that people are less likely to use this for getting hashes of workspace files.